corehq/apps/reports/standard/cases/basic.py

```diff
@@ -65,25 +65,30 @@ def _build_query(self):
         if self.case_status:
             query = query.is_closed(self.case_status == 'closed')
 
-        if self.request.can_access_all_locations and (
-                EMWF.show_all_data(mobile_user_and_group_slugs)
-                or EMWF.no_filters_selected(mobile_user_and_group_slugs)
-        ):
-            pass
+        case_owner_filters = []
 
-        elif (self.request.can_access_all_locations
-              and EMWF.show_project_data(mobile_user_and_group_slugs)):
-            query = query_all_project_data(
-                query, self.domain, mobile_user_and_group_slugs
-            )
+        if (
+            self.request.can_access_all_locations
+            and EMWF.show_project_data(mobile_user_and_group_slugs)
+        ):
+            case_owner_filters.append(all_project_data_filter(self.domain, mobile_user_and_group_slugs))
 
-        elif (self.request.can_access_all_locations
-              and EMWF.show_deactivated_data(mobile_user_and_group_slugs)):
-            query = query_deactivated_data(query, self.domain)
+        if (
+            self.request.can_access_all_locations
+            and EMWF.show_deactivated_data(mobile_user_and_group_slugs)
+        ):
+            case_owner_filters.append(deactivated_case_owners(self.domain))
 
-        else:  # Only show explicit matches
+        # Only show explicit matches
```

Comment:
I think this comment is now innaccurate.  That is, the report will show explicit matches in addition to other matches.